www/templates/testhistory/user.php

```diff
@@ -0,0 +1,90 @@
+<div class="history_hed">
+    <h1>Test History</h1>
+
+    <form name="filterLog" method="get" action="/testlog.php">
+
+        <?php if (!$is_logged_in) : ?>
+            <div class="logged-out-history">
+                <p>Test history is available for up to 30 days as long as your storage isn't cleared. By registering for a free account, you can keep test history for longer, compare tests, and review changes. Additionally, you will also be able to post on the <a href="https://forums.webpagetest.org">WebPageTest Forum</a> and contribute to the discussions there about features, test results and more.</p>
+                <a href="<?= "{$protocol}://{$host}/signup" ?>" class="btn-primary">Get Free Access</a>
+            </div>
+        <?php endif; ?>
+        <div class="history_filter">
+            <label for="filter">Filter test history:</label>
+            <input id="filter" name="filter" type="text" onkeyup="filterHistory()" placeholder="Search">
+            <?php if ($is_logged_in) : ?>
+                <label for="days" class="a11y-hidden">Select how far back you want to see</label>
+                <select name="days">
+                    <option value="1" <?php if ($days == 1) {
```

Comment:
I see this formatting and I think "I'm gonna end up using a lot of ternary operators to not see this formatting"